crypto/dh/dh_kdf.c

```diff
@@ -185,3 +190,4 @@ int DH_KDF_X9_42(unsigned char *out, size_t outlen,
     EVP_MD_CTX_cleanup(&mctx);
     return rv;
 }
+#endif
```

Comment:
@levitte I have been playing with the PEDANTIC flag and the debug-linux-x86_64 target but I was not able to produce a build failure on this branch. Do you have an example of how to test the effect of the missing dummy line?